### DISCUSSION TITLE
Error highlighting for mis-indented lines

### DISCUSSION THREAD

**Discussion by sf1tzp** [lukas-reineke/indent-blankline.nvim#560] [Ideas]
Hello! First off, thank you for creating this plugin, it's awesome 😃 

I used to use the [indent-rainbow](https://github.com/oderwat/vscode-indent-rainbow/) extension for VSCode. It offered an "ErrorColor" configuration, which would highlight mis-indented lines in red:

``` 
  // The indent color if the number of spaces is not a multiple of "tabSize".
  "indentRainbow.errorColor": "rgba(128,32,32,0.6)"
```

Here's an example of what that looks like in VS Code:
<img width="504" alt="image" src="https://user-images.githubusercontent.com/84053270/231223061-4ac96013-329a-4b81-bfce-9f4dce25e059.png">

Do you know if there is a way to achieve this with `indent-blankline`, or other vim/nvim configuration?

I used python as an example, but I find this most useful in large `yaml` files.

**Comment by lukas-reineke** [comment]
Interesting.
There is currently no way to do this, sorry.
Let's keep this issue open, I'll take a look at this when I have some time.
